tensorboard/components/tf_runs_selector/tf-runs-selector.html

```diff
@@ -50,7 +50,9 @@ <h3 id="tooltip-help" class="tooltip-container">Runs</h3>
     <tf-multi-checkbox
       id="multiCheckbox"
       names="[[runs]]"
+      selected-names="{{runSelectionState}}"
```

Comment:
I think perhaps "selectionState" is a better name for this property than "selectedNames" - otherwise it's confusing how it differs from "outSelected" which maps to "selectedRuns".  If we expect that in the context of run selection "run" == "name" then I would have thought "selectedRuns" and "selectedNames" would represent the same thing, but here they don't.

The name "selectionState" makes the distinction a bit clearer here while avoiding being run-specific.